cmd/serverless/main.go

```diff
@@ -384,9 +323,20 @@ func handleSignals(daemon *serverless.Daemon, stopCh chan struct{}) {
 		switch signo {
 		default:
 			log.Infof("Received signal '%s', shutting down...", signo)
-			daemon.Stop(false)
+			serverlessDaemon.Stop()
 			stopCh <- struct{}{}
 			return
 		}
 	}
 }
+
+func setupLogAgent(logChannel chan *logConfig.ChannelMessage) {
+	// we subscribed to the logs collection on the platform, let's instantiate
+	// a logs agent to collect/process/flush the logs.
```

Comment:
I would remove this comment, the function name is clear enough without it

Reply:
If we do want to document this function, we could add a docs comment above the function name instead.